src/Pim/Component/Catalog/Completeness/Checker/ChainedProductValueCompleteChecker.php

```diff
@@ -21,32 +31,43 @@ class ChainedProductValueCompleteChecker implements ProductValueCompleteCheckerI
      */
     public function isComplete(
         ProductValueInterface $productValue,
-        ChannelInterface $channel = null,
-        LocaleInterface $locale = null
+        ChannelInterface $channel,
+        LocaleInterface $locale
     ) {
         foreach ($this->productValueCheckers as $productValueChecker) {
-            if ($productValueChecker->supportsValue($productValue)
-                && !$productValueChecker->isComplete($productValue, $channel, $locale)
+            if ($productValueChecker->supportsValue($productValue, $channel, $locale)
+                && $productValueChecker->isComplete($productValue, $channel, $locale)
             ) {
-                return false;
+                return true;
             }
         }
 
-        return true;
+        return false;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function supportsValue(ProductValueInterface $productValue)
-    {
-        foreach ($this->productValueCheckers as $productValueChecker) {
-            if ($productValueChecker->supportsValue($productValue)) {
-                return true;
-            }
+    public function supportsValue(
```

Comment:
This method should be empty and return true. Those checks should be in the `isComplete` method.